metahtml/timestamp.py

```diff
@@ -379,6 +379,27 @@ def get_timestamp_published(html, url, **kwargs):
         ( 'time.com',                       '//div[contains(@class,"published-date")]' ),
         ( 'townhall.com',                   '//div[@class="contributor pull-left"][contains(.,"Posted:")]/text()'),
         ( 'voxeurop.eu',                    '//div[contains(@class,"publish_date_time")]' ),
+        ( 'today.it',                       '//time[@class="datestamp"]' ),
```

Comment:
All of these lines are supposed to be inserted so that the hostnames are sorted alphabetically.